jupyter-book/trajectories/pseudotemporal.ipynb

```diff
@@ -15,53 +15,95 @@
    ]
```

Comment:
<p>For some reason this cell is a level 2 header?!?</p><p> </p><p>"Even though samples are, traditionally, taken from a single experimental time point, a multitude of cell types can be observed. This diversity stems from the asynchronous nature of biological processes."</p><p>Not sure whether sentence two follows from sentence one. I'd argue that cell type diversity stems from the requirement of fulfilling different tasks in the body which have to be conducted by diverse, but specialized cells.</p><p> </p><h2>"developlemtal"</h2><p>typo</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/multimodal-single-cell-best-practices/pull/101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<blockquote>For some reason this cell is a level 2 header?!?</blockquote><p>I have no idea why it pops up as a level 2 header. Looks fine locally. Let's see what it looks like after I push again.</p><p> </p><blockquote>Not sure whether sentence two follows from sentence one. I'd argue that cell type diversity stems from the requirement of fulfilling different tasks in the body which have to be conducted by diverse, but specialized cells.</blockquote><p>Which is an asynchronous process, right? The sentence is simply saying that not all cells of a given cell type develop into another cell type at the same time. So I think it Is fine.</p>

Reply:
> I have no idea why it pops up as a level 2 header. Looks fine locally but still off after pushing my latest commits. Any idea what might be causing this mystery?

No, but don't worry about it.

> Which is an asynchronous process, right? The sentence is simply saying that not all cells of a given cell type develop into another cell type at the same time. So I think it Is fine.

Yeah, your statement here makes sense, but this not quite how I read your words to be honest. Maybe you can rephrase it a little bit to make it more clear, please? Not a big issue.